### PR TITLE
LPS-157848 [Raylife AP] Structure creation

### PR DESCRIPTION
**Pull request by eliasantosf**
Subtask: [LPS-157848](https://issues.liferay.com/browse/LPS-157848)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo.tsx [hunk 1]
```diff
@@ -12,202 +12,494 @@
...
+							<span className="text-danger-darken-1">*</span>
+						</label>
+					</div>
+
+					<div className="col filled form-condensed form-group">
+						<ClaySelect
+							aria-label="Select Label"
+							id="relationToContact"
+						>
+							{relationToContactOptions.map((item) => (
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo.tsx:291]: Please, change all .map parameter names.

```suggestion
							{relationToContactOptions.map((relationToContactOption) => (
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
LGTM :fire: But needs some adjustments.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo.tsx (hunk 1 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo.tsx]: Done!

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Pull request closed without merging**